Add labels to fetch package

jnicklas opened the pull request:
Closes #425

jnicklas committed:
Add labels to fetch package

Closes #425

cowboyd commented on packages/fetch/src/fetch.ts:
```diff
@@ -29,26 +23,33 @@ export function fetch(info: RequestInfo, requestInit: RequestInit = {}): Fetch {
 
   return {
+    name: `fetch \`${info}\``,
+    labels: {
+      expand: false
+    },
```
What about representing some of the `requestInit` inside the labels? It seems to me that parameters like `method` are extremely handy to have.

jnicklas replied:
Good idea!

jnicklas commented:
I added the request method in the labels and changed the naming scheme a bit, thinking we might adopt this naming scheme in other places (such as @effection/process) as well?

cowboyd reviewed (approved):
I like that naming scheme
